autopush/router/gcm.py

```diff
@@ -138,10 +142,18 @@ def _process_reply(self, reply):
         if len(reply.failed.items()) > 0:
             self.metrics.increment("updates.client.bridge.gcm.failed.failure",
                                    self._base_tags)
-            self.log.critical("GCM failures: {failed()}",
-                              failed=lambda: json.dumps(reply.failed.items()))
-            raise RouterException("GCM failure to deliver", status_code=503,
-                                  response_body="Please try request later.")
+            self.log.info("GCM failures: {failed()}",
+                          failed=lambda: json.dumps(reply.failed.items()))
+            self.router_table.register_user(
+                {"uaid": uaid_data.get('uaid'),
+                 "router_type": uaid_data.get("router_type", "gcm"),
+                 "connected_at": time.time(),
```

Comment:
Huh, we don't seem to be consistent about that. A quick grep shows that we've got time.time() in endpoint, ms_time() in websocket, I'll correct all of the instances to use ms_time.